test/ERC20CosmosCoinAmountConversionTest.t.sol

```diff
@@ -0,0 +1,490 @@
+        uint64 expectedConvertedAmount = 1000010; // 1,000,010 Cosmos coins
+        uint256 expectedRemainder = 0;
+
+        // Assertions
+        assertEq(convertedAmount, expectedConvertedAmount, "Converted amount mismatch");
+        assertEq(remainder, expectedRemainder, "Remainder mismatch");
+    }
+
+    function testConvertMockERC20MetadataAmountToCosmosCoin_7() public  {
+        uint8 decimals = 3; 
+        // Deploy the ERC20 token with metadata (custom decimals)
+        MockERC20Metadata customMockERC20Metadata = new MockERC20Metadata(decimals);
+        
+        uint256 amount = 1001; // 1.001 ERC20 tokens
+
+        // Call the conversion function
+    (uint64 convertedAmount, uint256 remainder) =
+    ERC20CosmosCoinAmountConversion._convertERC20AmountToCosmosCoin(address(customMockERC20Metadata), amount);
+
+        // Expected values for 18 decimals
+        uint64 expectedConvertedAmount = 1001000; // 1,000,010 Cosmos coins
+        uint256 expectedRemainder = 0;
+
+        // Assertions
+        assertEq(convertedAmount, expectedConvertedAmount, "Converted amount mismatch");
+        assertEq(remainder, expectedRemainder, "Remainder mismatch");
+    }
+
+    function testConvertMockERC20MetadataAmountToCosmosCoin_8() public  {
+        uint8 decimals = 1; 
+        // Deploy the ERC20 token with metadata (custom decimals)
+        MockERC20Metadata customMockERC20Metadata = new MockERC20Metadata(decimals);
+        
+        uint256 amount = 11; // 1.1 ERC20 tokens
+
+        // Call the conversion function
+    (uint64 convertedAmount, uint256 remainder) =
+    ERC20CosmosCoinAmountConversion._convertERC20AmountToCosmosCoin(address(customMockERC20Metadata), amount);
+
+        // Expected values for 18 decimals
+        uint64 expectedConvertedAmount = 1100000; // 1,000,010 Cosmos coins
+        uint256 expectedRemainder = 0;
+
+        // Assertions
+        assertEq(convertedAmount, expectedConvertedAmount, "Converted amount mismatch");
+        assertEq(remainder, expectedRemainder, "Remainder mismatch");
+    }
+
+    function testConvertMockERC20MetadataAmountToCosmosCoin_9() public  {
+        uint8 decimals = 0; 
+        // Deploy the ERC20 token with metadata (custom decimals)
+        MockERC20Metadata customMockERC20Metadata = new MockERC20Metadata(decimals);
+        
+        uint256 amount = 1; // 1 ERC20 tokens
+
+        // Call the conversion function
+    (uint64 convertedAmount, uint256 remainder) =
+    ERC20CosmosCoinAmountConversion._convertERC20AmountToCosmosCoin(address(customMockERC20Metadata), amount);
+
+        // Expected values for 18 decimals
+        uint64 expectedConvertedAmount = 1000000; // 1,000,010 Cosmos coins
+        uint256 expectedRemainder = 0;
+
+        // Assertions
+        assertEq(convertedAmount, expectedConvertedAmount, "Converted amount mismatch");
+        assertEq(remainder, expectedRemainder, "Remainder mismatch");
+    }*/
```

Comment:
I'd remove these test cases. We can access them through this git commit if we need them in the future anyway.
```suggestion
```

Reply:
If we squash and delete the branch, will the commit still be available?

Reply:
The merged code will contain the commit history, so yeah it should be available!